test_data/cql/expected_generated_queries/additional_temporal_disjoint_instant.rq

```diff
@@ -2,6 +2,9 @@ CONSTRUCT {
 ?focus_node <ex:updated_at> ?dt_1_instant
 }
 WHERE {
+
+FILTER EXISTS{
 ?focus_node <ex:updated_at> ?dt_1_instant .
 FILTER (?dt_1_instant > "2012-08-10T05:30:00+00:00"^^<http://www.w3.org/2001/XMLSchema#dateTime> || ?dt_1_instant < "2012-08-10T05:30:00+00:00"^^<http://www.w3.org/2001/XMLSchema#dateTime>)
+}
```

Comment:
Should these examples have the fallback triple added?
`?focus_node rdf:type ?prof_1_node_1`  
Or is this testing only the CQL generated filter, and the fallback triple gets added in `umbrella.py`?